common/lib/xmodule/xmodule/html_module.py

```diff
@@ -98,7 +98,7 @@ class HtmlModuleMixin(HtmlBlock, XModule):
         'js': [
             resource_string(__name__, 'js/src/collapsible.js'),
             resource_string(__name__, 'js/src/html/imageModal.js'),
-            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.pkgd.js'),
+            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.js'),
```

Comment:
OK, you don't have to sleuth. But I seriously doubt that it is needed by the HTML component (and question whether or not it would actually work).